addons/LegacyFatigue/functions/fnc_legacyFatigue.sqf

```diff
@@ -0,0 +1,159 @@
+/*
+ * Author: Original by BIS(Josef Zemanek), rewritten by PabstMirror
+ * Offers a fatigue system more like pre-1.54
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [player] call ace_legacyFatigue_fnc_legacyFatigue
+ *
+ * Public: No
+ */
+#define DEBUG_MODE_FULL
+#define ENABLE_PERFORMANCE_COUNTERS
+#include "script_component.hpp"
+
+#define TIME_FULL_CALC 0.5
+
+params ["_unit"];
+TRACE_1("params",_unit);
+
+_unit enableAimPrecision false;
+
+[{
+    BEGIN_COUNTER(all);
+    params ["_args", "_pfID"];
+    _args params ["_unit", "_lastFastRun", "_lastFullRun", "_scriptedChangePerSecond", "_scriptedChangeDone", "_lastPos", "_lastStamina", "_exhaustionEnd"];
+
+    if ((isNull _unit) || {!alive _unit} || {_unit != ACE_player}) exitWith {
+        TRACE_1("null/dead/notplayer",_unit);
+        [_pfID] call CBA_fnc_removePerFrameHandler;
+    };
```

Comment:
`!alive objNull` is `true` now. And `objNull == objNull` is `false`. Something seems redundant.